### ISSUE TITLE
Cannot output to different mount

### ISSUE THREAD

**Issue by popey456963** [mixn/carbon-now-cli#60]
`Rename` fails if the output directory is not on the same hard drive as the current working directory.  Here, /tmp/ is mounted to a different location as CWD:

```
carbon-now -h /tmp/tmp-31437-GSsSEGSH3nB0/code.js38fb969f-f483-496f-9b6c-5044718d76cc -t 38fb969f-f483-496f-9b6c-5044718d76cc
```

And it fails with:

```
5|web-file-uploader  | { [Error: EXDEV: cross-device link not permitted, rename '6b0c2b9e-842c-411f-a1b0-bed36024fc8f.png' -> '/tmp/tmp-31437-GSsSEGSH3nB0/code.js6b0c2b9e-842c-411f-a1b0-bed36024fc8f.png']
5|web-file-uploader  |   errno: -18,
5|web-file-uploader  |   code: 'EXDEV',
5|web-file-uploader  |   syscall: 'rename',
5|web-file-uploader  |   path: '6b0c2b9e-842c-411f-a1b0-bed36024fc8f.png',
5|web-file-uploader  |   dest:
5|web-file-uploader  |    '/tmp/tmp-31437-GSsSEGSH3nB0/code.js6b0c2b9e-842c-411f-a1b0-bed36024fc8f.png' }
```

**Comment by mixn** [comment]
As I’m finalizing the huge [`v2` release](https://github.com/mixn/carbon-now-cli/issues/83), I’ve decided to run a small beta of it. 🙂 This issue is covered in that release, so I’ll close it. 🙂 Deps updating is also automated from `v2` onwards.

To install the `v2` beta run

```
npm i carbon-now-cli@beta -g
```

[Docs 🚀 for `v2` can be found here.](https://github.com/mixn/carbon-now-cli/tree/v2/) 🙂 There are tons of new features and breaking changes, so make sure to re-read the docs if you’ve read them previously.

Happy testing and using!
`v2` will officially drop any day now. 🥳

**Comment by mixn** [comment]
@popey456963 I’ve commented above solely cause I’ve done so on all other issues addressed in #83. 

When it comes to the actual issue at hand here, I don’t know if it isn’t out of the scope of this tool simply cause

a) it hasn’t gotten a single upvote since 2020, so the demand doesn't seem to be large, and
b) I’ve personally never generated a screenshot with this CLI onto a different mount

Maybe you can check out of the error still exists in `v2` (it likely will) and then we can discuss things further afterwards.

Thanks in advance.
